### DISCUSSION TITLE
Does the server run the jobs on a schedule?

### DISCUSSION THREAD

**Discussion by zkvvoob** [immich-app/immich#1270] [Q&A] [answered]
Hello all,

I searched, but couldn't find any information about whether the server jobs (e.g. generate thumbmails) have to be triggered manually or if the server runs them at some intervals?

Thanks!

**Comment by bo0tzz** [comment, marked as answer]
There are a few jobs that are run daily:
* Thumbnail generation
* Video transcoding
* Reverse geocoding
* Exif extraction
* User deletion/cleanup

The relevant code that schedules them can be found at https://github.com/immich-app/immich/blob/main/server/apps/immich/src/modules/schedule-tasks/schedule-tasks.service.ts

**Reply by zkvvoob** [reply]
Thank you!

**Reply by jrasm91** [reply]
Just to follow up - the only jobs that are scheduled now are user deletion.

https://github.com/immich-app/immich/blob/main/server/apps/immich/src/app.cron-jobs.ts

**Comment by Migelo** [comment]
Is there a way to schedule the jobs to run daily (e.g. 3am) through the Admin UI?

**Reply by jrasm91** [reply]
No, there is not.

**Comment by twiclo** [comment]
Does this mean I need to manually go in to trigger the "Tag objects" and "Recognize faces" jobs?

**Reply by jrasm91** [reply]
Jobs run automatically when an image is uploaded, but you can always run them again via the admin jobs page for "missing".
